src/protocols/interaction_model/Constants.h

```diff
@@ -62,6 +62,49 @@ enum class MsgType : uint8_t
     TimedRequest          = 0x0a,
 };
 
+//TODO: Cleanup Reserved protocolCode from ember
+enum class ProtocolCode : uint16_t
+{
+    kSuccess                = 0x0,
+    kFailure                = 0x01,
+    kInvalidSubscription    = 0x7d,
+    kUnsupportedAccess      = 0x7e,
+    kUnsupportedEndpoint    = 0x7f,
+    kInvalidAction          = 0x80,
+    kUnsupportedCommand     = 0x81,
+    kReservedUnsuptedGeneralCommand   = 0x82,
+    kReservedUnsupManufClusterCommand = 0x83,
+    kReservedUnsupManufGeneralCommand = 0x84,
+    kInvalidCommand         = 0x85,
+    kUnsupportedAttribute   = 0x86,
+    kInvalidValue           = 0x87,
+    kUnsupportedWrite       = 0x88,
+    kResourceExhausted      = 0x89,
+    kReservedDuplicateExists = 0x8a,
+    kNotFound               = 0x8b,
+    kUnreportableAttribute  = 0x8c,
+    kInvalidDataType        = 0x8d,
+    kUnsupportedRead        = 0x8f,
+    kReservedInconsistentStartUpState = 0x90,
+    kReservedOutOfBand        = 0x91,
+    kReservedActionDenied              = 0x93,
+    kTimeout                = 0x94,
+    kReservedAbort                       = 0x95,
+    kReservedInvalidImage             = 0x96,
+    kReservedWaitForData               = 0x97,
+    kReservedNoImageAvailable          = 0x98,
+    kReservedRequiredMoreImage          = 0x99,
+    kReservedNotificationPending        = 0x9A,
```

Comment:
Can we be consistent about casing?  The others are lowercase...

Reply:
Done, thanks